python/ray/tune/examples/README.rst

```diff
@@ -52,6 +52,11 @@ LightGBM Example
 
 - `lightgbm_example <https://github.com/ray-project/ray/blob/master/python/ray/tune/examples/lightgbm_example.py>`__: Trains a basic LightGBM model with Tune with the function-based API and a LightGBM callback.
 
+:hugging_face: Transformers Example
```

Comment:
does this actually work? interesting

Reply:
I got it working for the sphinx docs but cant figure out how to do it for Github rst